offchain-modules/packages/scripts/src/stress-ckb-test.ts

```diff
@@ -0,0 +1,155 @@
+// your transfer ckb to recipients account privkey
+const ckbPrivateKey = '0xa6b8e0cbadda5c0d91cf82d1e8d8120b755aa06bc49030ca6e8392458c65fc80';
+const ethNodeUrl = 'http://127.0.0.1:8545';
+const ckbNodeUrl = 'http://127.0.0.1:8114';
+const ckbIndexerUrl = 'http://127.0.0.1:8116';
+const forceBridgeUrl = 'http://127.0.0.1:8080/force-bridge/api/v1';
+const ckbTokenAddress = '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
+const xchainTokenAddress = '0xe9B447cA594cB87B8d912040c8981B9696541B82';
+
+const lockCkbAmount = '30000000000000';
+const burnCkbSudtAmount = '10000000000000';
+
+// ts-node stress-test.ts [bridgeDirection] [batchNumber] [roundNumber]
+// bridgeDirection: lock = only test bridge lock, both = test bridge lock and burn
+// batchNumber: the number of lock/burn eth/erc20 txes sent one time
+// roundNumber: the number of batch round
+async function main() {
+  initLog({ level: 'debug', identity: 'stress-ckb-test' });
+  const bridgeDirection = nonNullable(process.argv[2]);
+  const batchNumber = Number(process.argv[3] ?? 100);
+  const roundNumber = Number(process.argv[4] ?? 2);
+
+  logger.info(
+    `start stress ckb test with bridgeDirection=${bridgeDirection}, batchNumber=${batchNumber}, roundNumber=${roundNumber}`,
+  );
+  const ckb = new CKB(ckbNodeUrl);
+  const client = new JSONRPCClient((jsonRPCRequest) =>
+    fetch(forceBridgeUrl, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+      },
+      body: JSON.stringify(jsonRPCRequest),
+      id: 1,
+    }).then((response) => {
+      if (response.status === 200) {
+        // Use client.receive when you received a JSON-RPC response.
+        return response.json().then((jsonRPCResponse) => client.receive(jsonRPCResponse));
+      } else if (jsonRPCRequest.id !== undefined) {
+        return Promise.reject(new Error(response.statusText));
+      }
+    }),
+  );
+
+  const provider = new ethers.providers.JsonRpcProvider(ethNodeUrl);
+
+  const ckbPrivateKeys = prepareCkbPrivateKeys(batchNumber);
+  const ckbAddresses = await prepareCkbAddresses(
+    ckb,
+    ckbPrivateKeys,
+    ckbPrivateKey,
+    ckbNodeUrl,
+    ckbIndexerUrl,
+    lockCkbAmount,
+  );
+  logger.info(`prepared ckb addresses ${ckbAddresses}`);
+
+  const ethWallets = await prepareEthWallets(provider, ethPrivateKey, batchNumber, 10000000000);
+  const recipients = ethWallets.map((ethWallet) => ethWallet.address);
+
+  logger.info('start stress lock test');
+  const stressPromise: PromiseLike<void>[] = [];
+  const lockPromise = stressLock(roundNumber, batchNumber, client, ckb, ckbPrivateKeys, recipients);
+  stressPromise.push(lockPromise);
+  await Promise.all(stressPromise);
+  if (bridgeDirection === 'both') {
+    const ckbAddress = ckb.utils.privateKeyToAddress(ckbPrivateKey, { prefix: AddressPrefix.Testnet });
+    await stressBurn(roundNumber, batchNumber, client, provider, ethWallets, ckbAddress);
+  }
```

Comment:
should stress lock and burn concurrently

Reply:
fixed